packages/noco-docs/content/en/setup-and-usages/formulas.md

```diff
@@ -97,6 +97,8 @@ Example: ({Column1} + ({Column2} * {Column3}) / (3 - $Column4$ ))
 |  |  | `DATEADD(date, 1, 'year')` | Supposing {DATE_COL} is 2022-03-14 03:14. The result is 2023-03-14 03:14. | DateTime columns and negative values are supported. Example: `DATEADD(DATE_TIME_COL, -1, 'year')` |
 |  |  | `IF(NOW() < DATEADD(date,10,'day'), "true", "false")` | If the current date is less than {DATE_COL} plus 10 days, it returns true. Otherwise, it returns false. | DateTime columns and negative values are supported. |
 |  |  | `IF(NOW() < DATEADD(date,10,'day'), "true", "false")` | If the current date is less than {DATE_COL} plus 10 days, it returns true. Otherwise, it returns false. | DateTime columns and negative values are supported. |
+| **DATETIME_DIFF** | `DATETIME_DIFF(date, date, ["millisecond" \| "ms" \| "second" \| "s" \| "minute" \| "mi" \| "hour" \| "hh" \| "day" \| "dy" \| "week" \| "wk" \| "month" \| "m" \| "quarter" \| "q" \| "year" \| "y"])` | `DATETIME_DIFF("2017/08/25", "2011/08/25", "seconds")` | Supposing {DATE_COL_1} is 2017-08-25 and {DATE_COL_2} is 2011-08-25. The result is -189388800. | Compares two dates and returns the difference in the unit specified. Positive integers indicate the second date being in the future compared to the first and vice versa for negative ones. |
+|  |  | `WEEKDAY(NOW(), "sunday")` | If today is Monday, it returns 1 | Get the week day of NOW() with the first day set as sunday |
```

Comment:
1. update the values
2. the result should be `189388800` instead because `date1` is greater than `date2`.

Reply:
Sorry but that's not the result of my testing can you please try it yourself with the dates in the same position as suggested here: https://www.w3schools.com/sql/trysqlserver.asp?filename=trysql_func_sqlserver_datediff

Reply:
Yes. I understand that is not result from the SQL. It's about how we display. As we are following Airtable behaviour, we should get the exact same result given the same formula.